cmd/ronin/config.go

```diff
@@ -161,6 +162,9 @@ func makeFullNode(ctx *cli.Context) (*node.Node, ethapi.Backend) {
 	}
 	backend, eth := utils.RegisterEthService(stack, &cfg.Eth)
 
+	// store internal transaction is enabled flag
+	rawdb.WriteStoreInternalTransactionsEnabled(backend.ChainDb(), ctx.GlobalBool(utils.StoreInternalTransactions.Name))
```

Comment:
In my opinion, it's better to add a field to blockchain object and read from it to check whether this feature is enabled instead of reading from DB

Reply:
it would be complicated to pass the config to blockchain object from cli and a lot of code changes. Therefore I think, the most simple wait is add it to db

Reply:
I've added a field `shouldStoreInternalTxs` in blockchain object, and get it from db to reduce validation cost everytine `WriteInternalTransactions` is triggered